### PR TITLE
refactor: rename project to ztpfw

### LINKED ISSUE #83: rename project name from code
rename project from code

### PR DESCRIPTION
**Pull request by iranzo**
# Description

Change References of kubeframe for ztpfw

Closes #83

### CODE REVIEW

File: shared-utils/common.sh [hunk 1]
```diff
@@ -137,9 +137,9 @@ export KUBECONFIG_HUB=${KUBECONFIG}
 echo ">>>> Grabbing info from Spokes File"
 echo ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
 
-export OC_DIS_CATALOG=kubeframe-catalog
+export OC_DIS_CATALOG=ztpfw-catalog
 export MARKET_NS=openshift-marketplace
-export KUBEFRAME_NS=kubeframe
+export ztpfw_NS=ztpfw
```
Comment (alknopfler) [inline shared-utils/common.sh:142]: Lower case?

File: ui/README.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ yarn build
 To create a container image with both backend and frontend:
 
 ```
-IMAGE=quay.io/mlibra/kubeframe:latest
+IMAGE=quay.io/mlibra/ztpfw:latest
```
Comment (alknopfler) [inline ui/README.md:47]: @mareklibra take a look with this image to be created before merged

Comment (alknopfler) [reply ui/README.md:47]: @iranzo ^^

File: ui/buildRunContainer.sh [hunk 3]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-IMAGE=quay.io/mlibra/kubeframe:latest
+IMAGE=quay.io/mlibra/ztpfw:latest
```
Comment (alknopfler) [inline ui/buildRunContainer.sh:3]: @iranzo ^^ @mareklibra ^^

File: ui/deployProd.sh [hunk 4]
```diff
@@ -1,14 +1,14 @@
 #!/bin/bash
-export IMAGE=${IMAGE:-'quay.io\/mlibra\/kubeframe:latest'}
-export ROUTE_HOST=${ROUTE_HOST:-"kubeframe.apps.mlibra-cim-04.redhat.com"}
-export NAMESPACE=${NAMESPACE:-"kubeframe-ui"}
+export IMAGE=${IMAGE:-'quay.io\/mlibra\/ztpfw:latest'}
+export ROUTE_HOST=${ROUTE_HOST:-"ztpfw.apps.mlibra-cim-04.redhat.com"}
```
Comment (alknopfler) [inline ui/deployProd.sh:3]: same comment

File: ui/frontend/public/index.html [hunk 5]
```diff
@@ -5,7 +5,7 @@
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
-    <title>Red Hat KubeFrame installer</title>
+    <title>Red Hat ztpfw installer</title>
```
Comment (alknopfler) [inline ui/frontend/public/index.html:8]: capital case

File: ui/frontend/src/components/DomainPage/DomainSelector.tsx [hunk 6]
```diff
@@ -17,7 +17,7 @@ export const DomainSelector: React.FC = () => {
         <Title headingLevel="h1">Do you want to use a custom domain?</Title>
       </StackItem>
       <StackItem className="wizard-sublabel">
-        Create unique URLs for your KubeFrame, such as device setup and console. <OptionalBadge />
+        Create unique URLs for your ztpfw, such as device setup and console. <OptionalBadge />
```
Comment (alknopfler) [inline ui/frontend/src/components/DomainPage/DomainSelector.tsx:20]: capital case

File: ui/frontend/src/components/DownloadSshKeyPage/DownloadSshKey.tsx [hunk 7]
```diff
@@ -70,7 +70,7 @@ export const DownloadSshKey: React.FC<{ setDownloaded: (isDownloaded: boolean) =
         <Alert
           variant={AlertVariant.warning}
           isInline
-          title="KubeFrame does not store the private key. Please download and keep your key in a safe place."
+          title="ztpfw does not store the private key. Please download and keep your key in a safe place."
```
Comment (alknopfler) [inline ui/frontend/src/components/DownloadSshKeyPage/DownloadSshKey.tsx:73]: capital case

File: ui/frontend/src/components/DownloadSshKeyPage/DownloadSshKey.tsx [hunk 8]
```diff
@@ -100,7 +100,7 @@ export const DownloadSshKey: React.FC<{ setDownloaded: (isDownloaded: boolean) =
             <img src={book} className="download-ssh__book" alt="Book icon" />
           </GridItem>
           <GridItem span={10}>
-            <div className="download-item__text">KubeFrame Private SSH Key</div>
+            <div className="download-item__text">ztpfw Private SSH Key</div>
```
Comment (alknopfler) [inline ui/frontend/src/components/DownloadSshKeyPage/DownloadSshKey.tsx:103]: capital case

File: ui/frontend/src/components/FinalPage/FinalPageSummary.tsx [hunk 9]
```diff
@@ -35,15 +35,15 @@ export const FinalPageSummary: React.FC = () => {
         </Text>
       </StackItem>
       <StackItem className="final-page-sumamary__item">
-        <Title headingLevel="h3">Enjoy your KubeFrame</Title>
+        <Title headingLevel="h3">Enjoy your ztpfw</Title>
```
Comment (alknopfler) [inline ui/frontend/src/components/FinalPage/FinalPageSummary.tsx:38]: capital case

File: ui/frontend/src/components/FinalPage/FinalPageSummary.tsx [hunk 10]
```diff
@@ -35,15 +35,15 @@ export const FinalPageSummary: React.FC = () => {
...
         <Text component={TextVariants.small}>
           MAC Your Kuberame has been successfully configured and is ready to use.
         </Text>
       </StackItem>
       <StackItem className="final-page-sumamary__item">
         <Title headingLevel="h3">Need to change something?</Title>
         <Text component={TextVariants.small}>
-          If you need to modify your settings, make sure you are still connected to your KubeFrame
+          If you need to modify your settings, make sure you are still connected to your ztpfw
```
Comment (alknopfler) [inline ui/frontend/src/components/FinalPage/FinalPageSummary.tsx:46]: same comments

File: ui/frontend/src/components/IngressIpPage/__snapshots__/IngressIpPage.test.tsx.snap [hunk 11]
```diff
@@ -64,7 +64,7 @@ exports[`IngressIpPage can render 1`] = `
       <div
         class="page-header-middle"
       >
-        KubeFrame setup
+        ztpfw setup
```
Comment (alknopfler) [inline ui/frontend/src/components/IngressIpPage/__snapshots__/IngressIpPage.test.tsx.snap:67]: capital case

File: ui/frontend/src/components/Page/Page.tsx [hunk 12]
```diff
@@ -6,7 +6,7 @@ import { HeaderTools } from './HeaderTools';
 import RedHatLogo from './RedHatLogo.svg';
 import './Page.css';
 
-const HeaderMiddlePart: React.FC = () => <div className="page-header-middle">KubeFrame setup</div>;
+const HeaderMiddlePart: React.FC = () => <div className="page-header-middle">ztpfw setup</div>;
```
Comment (alknopfler) [inline ui/frontend/src/components/Page/Page.tsx:9]: capital case

File: ui/frontend/src/components/PasswordPage/PasswordSelector.tsx [hunk 13]
```diff
@@ -46,7 +46,7 @@ export const PasswordSelector: React.FC<{
         </Title>
       </StackItem>
       <StackItem className="wizard-sublabel">
-        Keep your KubeFrame account safe and secure.
+        Keep your ztpfw account safe and secure.
```
Comment (alknopfler) [inline ui/frontend/src/components/PasswordPage/PasswordSelector.tsx:49]: capital case

File: ui/frontend/src/components/PersistPage/PersistPageBottom.tsx [hunk 14]
```diff
@@ -70,7 +70,7 @@ export const PersistPageBottom: React.FC = () => {
             />
           </StackItem>
           <StackItem isFilled className="wizard-sublabel">
-            Saving settings for your KubeFrame...
+            Saving settings for your ztpfw...
```
Comment (alknopfler) [inline ui/frontend/src/components/PersistPage/PersistPageBottom.tsx:73]: capital case

File: ui/frontend/src/components/WelcomePage/WelcomeBottom.tsx [hunk 15]
```diff
@@ -21,9 +21,9 @@ export const WelcomeBottom: React.FC<{ error?: string; nextPage?: string }> = ({
   return (
     <Stack className="welcome-bottom" hasGutter>
       <StackItem>
-        <Title headingLevel="h1">KubeFrame</Title>
+        <Title headingLevel="h1">ztpfw</Title>
```
Comment (alknopfler) [inline ui/frontend/src/components/WelcomePage/WelcomeBottom.tsx:24]: capital case

File: ui/package.json [hunk 16]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "KubeFrame",
+  "name": "ztpfw",
```
Comment (alknopfler) [inline ui/package.json:2]: same comment

File: ui/scripts/deployment.yaml [hunk 17]
```diff
@@ -67,11 +67,11 @@ spec:
         - name: CLUSTER_API_URL
           value: https://kubernetes.default.svc:443
         - name: OAUTH2_CLIENT_ID
-          value: "kubeframeoauth"
+          value: "ztpfwoauth"
```
Comment (alknopfler) [inline ui/scripts/deployment.yaml:70]: same comment

File: deploy-disconnected-registry/quay-manifests/config.yaml [hunk 18]
```diff
@@ -1,4 +1,4 @@
-REGISTRY_TITLE: Kubeframe Registry
+REGISTRY_TITLE: ztpfw Registry
```
Comment (alknopfler) [inline deploy-disconnected-registry/quay-manifests/config.yaml:1]: capital case

File: documentation/ztp-for-factories/ztp-for-factory-installation-workflow.adoc [hunk 19]
```diff
@@ -48,7 +48,7 @@ Resolving deltas: 100% (2182/2182), done.
 subscription.operators.coreos.com/openshift-pipelines-operator-rh unchanged
 >>>> Waiting for: Openshift Pipelines
 >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
->>>> Deploying Kubeframe Pipelines and tasks
+>>>> Deploying ztpfw Pipelines and tasks
```
Comment (alknopfler) [inline documentation/ztp-for-factories/ztp-for-factory-installation-workflow.adoc:51]: capital case

File: pipelines/bootstrap.sh [hunk 20]
```diff
@@ -108,7 +108,7 @@ function clone_ztp() {
 }
 
 function deploy_pipeline() {
-    echo ">>>> Deploying Kubeframe Pipelines Resources"
+    echo ">>>> Deploying ztpfw Pipelines Resources"
```
Comment (alknopfler) [inline pipelines/bootstrap.sh:111]: capital case

File: pipelines/resources/deploy-ztp-hub.yaml [hunk 21]
```diff
@@ -3,7 +3,7 @@ kind: Pipeline
 metadata:
   name: deploy-ztp-hub
 spec:
-  description: Tekton Pipeline to deploy Kubeframe Hub Cluster
+  description: Tekton Pipeline to deploy ztpfw Hub Cluster
```
Comment (alknopfler) [inline pipelines/resources/deploy-ztp-hub.yaml:6]: capital case

File: pipelines/resources/deploy-ztp-spokes.yaml [hunk 22]
```diff
@@ -3,7 +3,7 @@ kind: Pipeline
 metadata:
   name: deploy-ztp-spokes
 spec:
-  description: Tekton Pipeline to deploy Kubeframe Spokes clusters
+  description: Tekton Pipeline to deploy ztpfw Spokes clusters
```
Comment (alknopfler) [inline pipelines/resources/deploy-ztp-spokes.yaml:6]: capital case

**Review by alknopfler — CHANGES_REQUESTED** [review]
@iranzo please check it out

**Comment by alknopfler** [comment]
@mareklibra please check it out

**Comment by mareklibra** [comment]
Minor fixes added to #80 and #86, they are merged now

**Review by alknopfler — COMMENTED** [review]
By now, LGTM, waiting new changes

**Comment by mareklibra** [comment]
lgtm with later patch fixing UI unit tests

**Comment by mareklibra** [comment]
https://quay.io/repository/ztpfw/ui?tab=tags is updated now

**Review by alknopfler — APPROVED** [review]

**Pull request merged**